<!doctype html>
<html>
	<head>
		<meta charset="utf-8">
		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">

		<title>BIONF tools presentation</title>

		<link rel="stylesheet" href="css/reveal.css">
		<link rel="stylesheet" href="css/theme/simple.css" id=theme>

		<!-- Theme used for syntax highlighting of code -->
		<link rel="stylesheet" href="lib/css/zenburn.css">

		<!-- Printing and PDF exports -->
		<script>
			var link = document.createElement( 'link' );
			link.rel = 'stylesheet';
			link.type = 'text/css';
			link.href = window.location.search.match( /print-pdf/gi ) ? 'css/print/pdf.css' : 'css/print/paper.css';
			document.getElementsByTagName( 'head' )[0].appendChild( link );
		</script>
	</head>
	<body>
		<div class="reveal">
			<div class="slides">

				<!-- SLIDE -1 title -->
				<section data-background-image="images/00_logo.png" data-background-position="50% 90%" data-background-size="15%">
					<h2>THE BIONF UNIVERSE</h2>
					<hr>
					<h4>22.01.2026</h4>
					<h4>Vinh Tran</h4>
				</section>

				<!-- SLIDE 0 About-me -->
				<section>
					<h5 style="color: white; background-color: #559951"><strong>ABOUT ME</strong></h5>
					<section data-transition="none">
						<img src="images/00_aboutme.jpg" style='float: top; border:0; box-shadow: none' height=590/>
						<aside class="notes">
							1) blabalbla<br><br>
							2) blablabla<br><br>
							3) Dynamic evolution (gene set reduction or expansion)<br><br>
							4) After my PhD, I stayed in the group and was involved in several projects<br>
							developing algorithm and tools for comparative genomics analyses
						</aside>
					</section>
				</section>

				<!-- SLIDE 1 terms -->
				<section>
					<h5 style="color: white; background-color: #559951"><strong>KEYWORDS</strong></h5>
					<section data-transition="none">
						<img src="images/013_comparative_genomics.png" style='float: top; border:0; box-shadow: none' height=500/>
						<aside class="notes">
							Comparative genomics is a field of biological research in which
							one compares the genomic sequences and characteristics of multiple organisms
							in order to find out the similarity and difference between them.
						</aside>
					</section>
					<section data-transition="none">
						<p>Homologs = Orthologs ∪ Paralogs</p>
						<img src="images/012_homology.png" style='float: top; border:0; box-shadow: none' height=300/>
						<aside class="notes">
							Homologous genes = genes that share a common ancestor<br>
							Orthologs arise from a common ancester gene after a speciation event,
							leading to similar genes (hence potentially conserved function)<br>
							While, the result of a duplication event are paralogs that can evolve
							new functions
						</aside>
					</section>
					<section data-transition="none">
						<img src="images/011_terms.png" style='float: top; border:0; box-shadow: none' height=500/>
						<aside class="notes">
							These are some other keywords, for which I will explain during my presentation
						</aside>
					</section>
				</section>

				<!-- SLIDE 2 orthology -->
				<section>
					<h5 style="color: white; background-color: #559951"><strong>QUESTION</strong></h5>
					<section data-transition="none">
						<br>
						<img src="images/021_newspecies.png" style='float: top; border:0; box-shadow: none' height=440/>
						<p>What genetic mechanisms underlie this fish’s<br>unique phenotype compared to related species?</p>
						<aside class="notes">
							Let start with a scientific question. So, we observed something interest about an organism,
							so we ask ourselve, what make this organism different than the others (closely related species)?
							Which genetic mechanisms give this species such a unique phenotype?
							More precise, which regulatory changes contribute to this observation,
							or, which genes or proteins are responsible for this phenotype, etc.
						</aside>
					</section>
				</section>

				<!-- SLIDE 3 Sequencing -->
				<section>
					<h5 style="color: white; background-color: #559951"><strong>SEQUENCING</strong></h5>
					<section data-transition="none">
						<br>
						<img src="images/031_sequencing.gif" style='float: top; border:0; box-shadow: none' height=400/>
						<aside class="notes">
							The next step we should do is to take a sample of the organism and do the sequencing.
							Why:
							- With the current technique, it is the cheapest (maximal info per dolar) and quickest way to have
							an overview of the novelty of the organism such as: identify gene gain/loss,
							gene family expansion, unique or fast-evolving genes, compare regulatory regions,
							infer metabolic pathways, so on and so forth<br>
							- And therefore we can better formulate our hypothesis for the downstream analyses
						</aside>
					</section>
					<section data-transition="none">
						<br>
						<img src="images/032_assembly.jpg" style='float: top; border:0; box-shadow: none' height=600/>
						<aside class="notes">
							This is a standard pipeline of any sequencing project.<br>
							First, we cut the original genome into pieces to put into the sequencing machine and get out milions of sequence reads<br>
							Then, we assemble those reads into larger fragments called contigs and scaffolds<br>
							If we are lucky, we can reconstruct the whole chromosome or original genome<br>
							After that, we do the genome annotation to predict genes and gene products from the assembly
						</aside>
					</section>
					<section data-transition="none">
						<br>
						<img src="images/033_assembly_2.jpg" style='float: top; border:0; box-shadow: none' height=600/>
						<aside class="notes">
							This assembly and the gene set or protein set are the fundamental data for studying our species of interest.<br>
							Because of that, the quality of these data is very important and can affect all the conclusion of our downstream analyses
						</aside>
					</section>
				</section>

				<!-- SLIDE 4 Quality control -->
				<section>
					<h5 style="color: white; background-color: #559951"><strong>QUALITY CONTROL</strong></h5>
					<section data-transition="none">
						<img src="images/041_data.jpg" style='float: top; border:0; box-shadow: none' height=200/>
						<img src="images/042_quality.jpg" style='float: top; border:0; box-shadow: none' height=300/>
						<aside class="notes">
							Therefore, the first step before doing any further studies is to check for the quality of our assembly and gene annotation<br>
							There are some basic metrics for evaluating the quality of an assembly, such as N50 length, the genome coverage, BUSCO completeness, etc.<br>
							And from this step come our BIONF tools into the game
						</aside>
					</section>
					<section data-visibility="hidden">
						<br>
						<img src="images/041_data.jpg" style='float: top; border:0; box-shadow: none' height=200/>
						<p>
							Is my genome "clean"?
							<p style="font-size:30px; text-align:center">Contamination</p>
							<p style="font-size:30px; text-align:center">Horizontal gene transfer</p>
						</p>
						<img src="images/043_taxaminer.jpg" style='float: top; border:0; box-shadow: none' height=150/>
						<aside class="notes">
							One of the question one can ask after having the assembly is that, is it clean of contamination or HGT?<br>
							Because contaimination can lead to a wrong intepretation. Such as:<br>
							Having chicken gene in the fish assembly doesn't automatically mean that our fish has something to do with chicken<br>
							It could just be a side effect of someone eating KFC while doing the sequencing...<br>
							To find out such contamination genes, one of my colleague, Freya, has developed a tool called taXaminer<br><br>
						</aside>
					</section>
					<section data-transition="none">
						<br>
						<img src="images/044_taxaminer_workflow.png" style='float: top; border:0; box-shadow: none' height=300/>
						<p style="font-size:12px; text-align:right"><em>Freya Arthen (2022), Master thesis</em></p>
						<p><a href="https://github.com/BIONF/taXaminer" target="_blank">https://github.com/BIONF/taXaminer</a></p>
						<aside class="notes">
							taXaminer will have the assembly data as the main input and take into account many characters to cluster the similar genes together,
							such as the read coverage, the GC content, the gene length, the gene position on the contigs, etc.<br>
							It uses the Principal component analysis (PCA) to reduce the dimention of those characters into 2 or 3 dimentions
							in order to be easy to show and intepret with 2D or 3D plot<br><br>
							For example here in this small cloud, we see many Ascomycota hits.
							They are all belong to short scaffolds and have low read coverage in compare to the others.
							Therefore, one could consider them as candidates for contamination.
						</aside>
					</section>
					<section data-transition="none">
						<br>
						<img src="images/041_data.jpg" style='float: top; border:0; box-shadow: none' height=200/>
						<p>
							Is my gene set complete? And <span style="text-decoration: underline;"><strong><em>how</em></strong></span> complete is it?
						</p>
						<img src="images/046_fcat_logo.png" style='float: top; border:0; box-shadow: none' height=100/>
						<aside class="notes">
							Another question we can ask: Is my gene set complete? This can be answered with the most well-known tool BUSCO.
							But, we want to know more, which is how complete, or how fragmented is my gene set?<br>
							To adress this question, we have developed fCAT
						</aside>
					</section>
					<section data-transition="none">
						<br>
						<img src="images/047_fcat_workflow.png" style='float: top; border:0; box-shadow: none' height=400/>
						<p style="font-size:12px; text-align:right"><em>Tran & Ebersberger (2022),
							<a href="https://doi.org/10.7490/f1000research.1119126.1" target="_blank">https://doi.org/10.7490/f1000research.1119126.1</a></em></p>
						<p><a href="https://github.com/BIONF/fCAT" target="_blank">https://github.com/BIONF/fCAT</a></p>
						<aside class="notes">
							The starting point of fCAT is the core set for each taxonomic clade
							(eukaryota, metazoa, sarcopterygii, actinopterygii, lecanoromyces, alveolata).
							fCAT core sets are currently created by using OMA ortholog groups,
							but they can be replaced by any other ortholog groups like from OrthoMCL, OrthoDB, HieranoiDB,...<br>
							fCAT will search ortholog for these core groups in the species of interest using a targeted search algorithm,<br>
							then classify the founded orthologs as complete or fragmented based on the sequence length,
							or as similar or dissimilar based on the protein domain architectures
						</aside>
					</section>
					<section data-transition="none">
						<br>
						<img src="images/048_fcat_example.png" style='float: top; border:0; box-shadow: none' height=600/>
						<aside class="notes">
							Different than BUSCO, which can only assign a gene as complete or fragmented, fCAT delivers more information<br>
							Such as in this example, we found an "fragmented" ortholog in the species of interest, the Trachinus draco species.
							It lacks the BTB and BACK domains in the N-terminus im comparison to the orthologs of other species.
							By looking at the assembly, we found a direct upstream gene, which has exactly these 2 missing domains.
							So, this could be a fission event that happened in Trachinus draco which split the original gene into 2 parts.
						</aside>
					</section>
				</section>

				<!-- SLIDE 5 Targeted ortholog search -->
				<section>
					<h5 style="color: white; background-color: #559951"><strong>Orthology search</strong></h5>
					<section data-transition="none">
						<br>
						<img src="images/051_paper.png" style='float: left; border:0; box-shadow: none' height=200/>
						<img src="images/052_paper.png" style='float: right; border:0; box-shadow: none' height=200/>
						<aside class="notes">
							After validating the genome assembly and annotation, we address our original question<br>
							by examining a key functional trait: the ability to synthesize DHA.<br>
							For that, we search through literatures for the same study in other organisms to get the reference data
						</aside>
					</section>
					<section data-transition="none">
						<br>
						<img src="images/053_fads_protein.jpg" style='float: top; border:0; box-shadow: none' height=100/>
						<p>Feature-aware directed (targeted) ortholog search</p>
						<img src="images/053b_fdog_ms.png" style='float: top; border:0; box-shadow: none' height=150/>
						<img src="images/053_fdog_logo.png" style='float: top; border:0; box-shadow: none' height=100/>
						<aside class="notes">
							Here, the main enzymes that involved in the DHA biosynthesis pathway are the FADs and ELOVL proteins.<br>
							The straighforward approach here is searching for these genes in our species.
							For that, we use the targeted ortholog search tool fDOG.<br>
							<br>FAD: Flavin adenine dinucleotide => enzyme for fatty acid desaturation (remove 2 Hydrogen atoms to create carbon/carbon double bond)
							<br>ELOVL: Elongase of very long-chain fatty acid
						</aside>
					</section>
					<section data-transition="none">
						<br>
						<img src="images/054_fdog_workflow.jpg" style='float: top; border:0; box-shadow: none' height=470/>
						<p style="font-size:12px; text-align:right"><em>Tran et al. (2025),
							<a href="https://doi.org/10.1093/molbev/msaf120" target="_blank">https://doi.org/10.1093/molbev/msaf120</></em></p>
						<p><a href="https://github.com/BIONF/fDOG" target="_blank">https://github.com/BIONF/fDOG</a></p>
						<aside class="notes">
							Starting with a seed protein (the FAD or ELOVL from the previous slide),
							fDOG iteratively search for orthologs in a set of so-called core taxa. After each round,
							new ortholog will be added to the core ortholog group to create a profile HMM for the core group.
							When the core group reach to a pre-defined number of orthologs, this core compilation step will stop and the final pHMM will be created.<br>
							Then, fDOG use this pHMM to search for similar sequences in the species of interest.
							The candidate HMM hits will then be checked, if it passes the reciprocal criterium, it will be considered as an ortholog
							and will be added to the final ortholog group.<br>
							Finally, fDOG create a phylogenetic profiles for the seed protein with the founded orthologs in different taxa
						</aside>
					</section>
				</section>

				<!-- SLIDE 6 Phylogenetic profile analysis -->
				<section>
					<h5 style="color: white; background-color: #559951"><strong>Phylogenetic profile analysis</strong></h5>
					<section data-transition="none">
						<br>
						<p>Phylogenetic profile</p>
						<img src="images/060_pp_simple.png" style='float: top; border:0; box-shadow: none' height=300/>
						<p style="font-size:30px">Presence of orthologs ⇒ likely presence of the function</p>
						<aside class="notes">
							The main purpose of the phylogenetic profiles from fDOG is not different than showing the presence/absence pattern of the seed proteins in a list of diverse species<br>
							The only different from the original phylogenetic profiles is,
							that the one from fDOG is not binary but it contains also other measures,
							namely the domain architecture similarity between the seed and the orthologs.<br>
							To visualize and explore such complex phylogenetic profiles, an excel table is not always useful
						</aside>
					</section>
					<section data-transition="none">
						<br>
						<p>Phylogenetic profile</p>
						<img src="images/060b_pp_simple2.png" style='float: top; border:0; box-shadow: none' height=300/>
						<p style="font-size:30px">Similar profiles ⇒ functional association</p>
						<aside class="notes">
							The main purpose of the phylogenetic profiles from fDOG is not different than showing the presence/absence pattern of the seed proteins in a list of diverse species<br>
							The only different from the original phylogenetic profiles is,
							that the one from fDOG is not binary but it contains also other measures,
							namely the domain architecture similarity between the seed and the orthologs.<br>
							To visualize and explore such complex phylogenetic profiles, an excel table is not always useful
						</aside>
					</section>
					<section data-transition="none">
						<br>
						<p>Visualization and exploration of phylogenetic profiles</p>
						<br>
						<img src="images/061b_phyloprofile_ms.png" style='float: top; border:0; box-shadow: none' height=150/>
						<img src="images/061_phyloprofile_logo.png" style='float: top; border:0; box-shadow: none' height=100/>
						<p style="font-size:12px; text-align:right"><em>Tran et al. (2018),
							<a href="https://doi.org/10.1093/bioinformatics/bty225" target="_blank">https://doi.org/10.1093/bioinformatics/bty225</a></em><br>
						<p style="font-size:12px; text-align:right"><em>Tran & Ebersberger (2025),
							<a href="https://doi.org/10.48550/arXiv.2504.19710" target="_blank">https://doi.org/10.48550/arXiv.2504.19710</a></em><br>
						<aside class="notes">
							....<br>
							Therefore we have developed this PhyloProfile tool
						</aside>
					</section>
					<section data-transition="none">
						<br>
						<img src="images/062_phyloprofile_features.png" style='float: top; border:0; box-shadow: none' height=500/>
						<p><a href="https://github.com/BIONF/PhyloProfile" target="_blank">https://github.com/BIONF/PhyloProfile</a></p>
						<aside class="notes">
							I think some of you have already known about PhyloProfile.<br>
							It has an interactive phylogenetic profile plot,
							in which each dot in the plot show the presence of the ortholog for the corresponding taxon,
							and the color of the dot or the color of the background represent the value of the protein architecture similarity<br>
							With PhyloProfile we can not only viewing the comparison of the protein architectures,
							but we can also perform some downstream analysis, such as estimate the gene age based on the
							last common ancestor approach; or identify group of core genes that can be used fo a species tree reconstruction
						</aside>
					</section>
				</section>

				<!-- SLIDE 7 FAS -->
				<section>
					<h5 style="color: white; background-color: #559951"><strong>Protein feature comparison</strong></h5>
					<section data-transition="none">
						<br>
						<img src="images/063_domain_comparison.png" style='float: top; border:0; box-shadow: none' height=400/>
						<p>Protein feature architecture comparison</p>
						<aside class="notes">
							The most relevant feature is the comparison of the protein domain architecture.<br>
							With this, we can easily find genes that have changes in their architectures, which suggest also for
							functional changes. Or, we can figure out, if the difference in the domain architecture is just
							a technical artifact due to wrong gene prediction.<br>
							For example here is the domain plot of a human protein agains 2 orthologous proteins from 2 different Chimpanzee gene sets.
							We see the ortholog in the QfO gene set is much shorter than the one in the NCBI Refseq gene set.
							It is missing the whole path in the N-terminus. So, our hypothesis is a wrong gene prediction happened in the QfO dataset.<br>
							<em>(The possibility for a gene fission event here is rather low. Because, human and chimp are very close relative.
							If the fission event happened in Chimp, we should also see that in human.
							And if it is the case, then we had 2 independent wrong gene prediction in human gene set and in the chimpanzee refseq gene sets.
							Which is quite hard to believe)</em>
						</aside>
					</section>
					<section data-transition="none">
						<br>
						<p>Feature architecture similarity (FAS)</p>
						<img src="images/072b_FAS_ms.png" style='float: top; border:0; box-shadow: none' height=150/>
						<img src="images/072_julian.jpg" style='float: top; border:0; box-shadow: none' height=180/>
						<aside class="notes">
							To sytematically compare the protein architectures by not just looking at the domain plot,
							we have come up with a new metric called the FAS score<br>
							Currently, the FAS scores are calculated using the greedyFAS tool, developed by Julian
						</aside>
					</section>
					<section data-transition="none">
						<br>
						<img src="images/073_FAS.png" style='float: top; border:0; box-shadow: none' height=200/>
						<p style="font-size:12px; text-align:right"><em>Dosch et al. (2018),
							<a href="https://doi.org/10.1093/bioinformatics/btad226" target="_blank">https://doi.org/10.1093/bioinformatics/btad226</a></em><br>
						<p><a href="https://github.com/BIONF/FAS" target="_blank">https://github.com/BIONF/FAS</a></p>
						<aside class="notes">
							FAS score has a value between 1 and 0; 1: completely similar; 0: completely different or no annotated domain found.<br>
							Given 2 proteins, greedyFAS will annotate them with different features such as PFAM/SMART domains, transmembrane domains, low complexity regions,...
							Then, the tool compares the 2 annotated architectures and penalize the missing features, depending on which protein is considered as the reference protein.<br>
							For example here, if we take A as the reference and compare with B, the Pleckstrin domain is missing, therefore the FAS score in this direction is only 0.6<br>
							But, in the reverse direction, all the domains of B can be found in A, which will lead to a maximum FAS score of 1.
						</aside>
					</section>
				</section>

				<!-- SLIDE 8 EXAMPLES -->
				<section>
					<h5 style="color: white; background-color: #559951"><strong>Use cases</strong></h5>
					<section data-transition="none">
						<br>
						<div style="display: flex; align-items: center; justify-content: center; gap: 20px;">
					    <img src="images/085_example1a.png"
					         style="border:0; box-shadow:none;"
					         height="200">
					    <img src="images/085_example1.png"
					         style="border:0; box-shadow:none;"
					         height="320">
					  </div>
						<p style="font-size:12px; text-align:right"><em>Reproduced from Figure 1, Moretti et al. (2017),
							<a href="https://doi.org/10.1038/s41598-017-16947-5" target="_blank">https://doi.org/10.1038/s41598-017-16947-5</a></em><br>
						<aside class="notes">
							...
						</aside>
					</section>
					<section data-transition="none">
						<br>
						<div style="display: flex; align-items: center; justify-content: center; gap: 20px;">
					    <img src="images/086_example2a.png"
					         style="border:0; box-shadow:none;"
					         height="200">
					    <img src="images/086_example2.png"
					         style="border:0; box-shadow:none;"
					         height="450">
					  </div>
						<p style="font-size:12px; text-align:right"><em>Reproduced from Figure 8, Birikmen et al. (2021),
							<a href="https://doi.org/10.3389/fmicb.2021.739000" target="_blank">https://doi.org/10.3389/fmicb.2021.739000</a></em><br>
						<aside class="notes">
							...
						</aside>
					</section>
					<section data-transition="none">
						<br>
						<div style="display: flex; align-items: center; justify-content: center; gap: 20px;">
					    <img src="images/087_example3a.png"
					         style="border:0; box-shadow:none;"
					         height="150">
					    <img src="images/087_example3.png"
					         style="border:0; box-shadow:none;"
					         height="350">
					  </div>
						<p style="font-size:12px; text-align:right"><em>Reproduced from Figure 4, Tran et al. (2025),
							<a href="https://doi.org/10.1093/molbev/msaf120" target="_blank">https://doi.org/10.1093/molbev/msaf120</a></em><br>
						<aside class="notes">
							...
						</aside>
					</section>
				</section>

				<!-- SLIDE 9 Summary -->
				<section>
					<h5 style="color: white; background-color: #559951"><strong>Summary</strong></h5>
					<section data-transition="none">
						<br>
						<img src="images/111_summary.jpg" style='float: top; border:0; box-shadow: none' height=620/>
						<aside class="notes">
							As a summary, here I show you the relationship between our tools.<br>
							The grey arrows shows the input and output flow between the tools;
							the blue arrows shows the integration of one tool in the others.<br>
							Most of the tools are written in Python and are available as python packages.<br>
							For the graphical user interface tools, we have different approaches,
							from JavaScript for MATT or R-Shiny for PhyloProfile.<br>
							Beside the ability to install the tools from PyPi or Bioconductor,
							you can get the source code of those tools on the BIONF github repository
						</aside>
					</section>
				</section>

				<!-- SLIDE 12 links-->
				<section>
					<h5 style="color: white; background-color: #559951"><strong></strong></h5>
					<section>
						<img src="images/121_thanks.jpeg" style='float: top; border:0; box-shadow: none' height="250"/>
						<br>
						<p style="font-size:15px; text-align:center"><em>Presentation link</em><br>
						<a href="https://trvinh.github.io/bionf_universe_me/slides/#/" target="_blank">https://trvinh.github.io/bionf_universe_me/slides/#/</a></p>
						<img src="images/122_qrcode.png" style='float: top; border:0; box-shadow: none' height="100"/>
					</section>
				</section>

			</div>
		</div>

		<script src="lib/js/head.min.js"></script>
		<script src="js/reveal.js"></script>

		<script>
			// More info about config & dependencies:
			// - https://github.com/hakimel/reveal.js#configuration
			// - https://github.com/hakimel/reveal.js#dependencies
			Reveal.initialize({
				controls: true,
				progress: true,
				history: true,
				center: true,
				backgroundTransition: 'zoom', // Not working?!?!?

				dependencies: [
					{ src: 'lib/js/classList.js', condition: function() { return !document.body.classList; } },
		      { src: 'plugin/markdown/marked.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
		      { src: 'plugin/markdown/markdown.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
		      { src: 'plugin/zoom-js/zoom.js', async: true, condition: function() { return !!document.body.classList; } },
		      { src: 'plugin/notes/notes.js', async: true, condition: function() { return !!document.body.classList; } }
				]
			});
		</script>
	</body>
</html>
